CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.7.0-RC3
+
+- IDP User Migration
+  - Error Handling Logic
```

Comment:
Lets be a bit more specific - something like that:

- extended error handling of the api response message implemented
- included csv file upload validation of csv header row content